tests/SSW.Rewards.UnitTests/Sut/ViewModels/OnBoardingViewModelTests.cs

```diff
@@ -0,0 +1,236 @@
+        Sut.Items[3].HasPoints.Should().BeTrue();
+        Sut.Items[3].Points.Should().Be(2000);
+        Sut.Items[3].IsAnimation.Should().BeFalse();
+    }
+    
+    [Fact]
+    public void Ctor_ShouldInitializeMiBandCarouselItem()
+    {
+        // Arrange
+
+        // Act
+
+        // Assert
+        Sut.Items[4].Content.Should().Be("Get on the leaderboard and earn a Mi Wrist band. Just like a FitBit, except more functionality and a month's battery life!");
+        Sut.Items[4].Image.Should().Be("v2band");
+        Sut.Items[4].SubHeading.Should().Be("Mi Band 6");
+        Sut.Items[4].ButtonText.Should().Be("NEXT");
+        Sut.Items[4].HasPoints.Should().BeTrue();
+        Sut.Items[4].Points.Should().Be(6000);
+        Sut.Items[4].IsAnimation.Should().BeFalse();
+    }
+    
+    [Fact]
+    public void Ctor_ShouldInitializeSpecificationReviewCarouselItem()
+    {
+        // Arrange
+
+        // Act
+
+        // Assert
+        Sut.Items[5].Content.Should().Be("SSW Architects will help you successfully implement your project.");
+        Sut.Items[5].Image.Should().Be("v2consultation");
+        Sut.Items[5].SubHeading.Should().Be("Half Price Specification Review");
+        Sut.Items[5].ButtonText.Should().Be("DONE");
+        Sut.Items[5].HasPoints.Should().BeTrue();
+        Sut.Items[5].Points.Should().Be(3000);
+        Sut.Items[5].IsAnimation.Should().BeFalse();
+    }
+
+    [Fact]
+    public void Ctor_ShouldSetSelectedItemToFirst()
+    {
+        // Arrange
+        var expected = Sut.Items.First();
+        
+        // Act
+
+        // Assert
+        Sut.SelectedItem.Should().Be(expected);
+    }
+
+    [Fact]
+    public void Ctor_ShouldSetDetailsFromSelectedItem()
+    {
+        // Arrange
+        var selectedItem = Sut.SelectedItem;
+
+        // Act
+
+        // Assert
+        Sut.SubHeading.Should().Be(selectedItem.SubHeading);
+        Sut.Content.Should().Be(selectedItem.Content);
+        Sut.ButtonText.Should().Be(selectedItem.ButtonText);
+        Sut.HasPoints.Should().Be(selectedItem.HasPoints);
+        Sut.Points.Should().Be(selectedItem.Points);
+    }
+    
+    #endregion
+    
+    #region DoActionCommand
+
+    [Fact]
+    public void DoActionCommand_ShouldCallPopModalAsync_WhenIsLastItemTrue()
+    {
+        // Arrange
+        Sut.Navigation = Mocker.GetMock<INavigation>().Object;
+        Sut.SelectedItem = Sut.Items.Last();
+
+        // Act
+        Sut.DoActionCommand.Execute(null);
+
+        // Assert
+        Mocker.GetMock<INavigation>().Verify(c => c.PopModalAsync());
+    }
+
+    [Fact]
+    public void DoActionCommand_ShouldRaiseScrollToRequestedEvent_WhenIsLastItemFalse()
+    {
+        // Arrange
+        var nextSelectedIndex = Sut.Items.IndexOf(Sut.SelectedItem) + 1;
+        
+        // Act
+        var raisedEvent = Assert.Raises<int>(handler => Sut.ScrollToRequested += handler, handler => Sut.ScrollToRequested -= handler,
```

Comment:
For some reason didn't manage to get it working with `FluentAssertions` 😕 With `Assert` it works fine, but if anyone knows how to do that via `FluentAssertions` - please let me know.